storage/redis/util.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package redis
+
+func escapeMatch(match []byte) []byte {
```

Comment:
this worries me. does redis not have its own escape stuff already?

Reply:
I failed to find one. I even failed to find a proper spec for the matcher syntax.